src/cljs/main/broadfcui/common/sync.cljs

```diff
@@ -0,0 +1,103 @@
+(ns broadfcui.common.sync
+  (:require
+   [dmohs.react :as react]
+   [clojure.string :as string]
+   [broadfcui.common.components :as comps]
+   [broadfcui.common.style :as style]
+   [broadfcui.components.modals :as modals]
+   [broadfcui.endpoints :as endpoints]
+   [broadfcui.utils :as utils]
+   ))
+
+
+(defn get-method-display [{:keys [namespace name snapshotId]}]
+  (str namespace "/" name " snapshot " snapshotId))
+
+
+(react/defc SynchronizeModal
+  {:render
+   (fn [{:keys [props state this]}]
+     [modals/OKCancelForm
+      {:header "Synchronize Access to Method"
+       :dismiss (:dismiss props)
+       :content
+       (react/create-element
+        [:div {:style {:maxWidth 670}}
+         [:div {:style {:marginBottom "1.5rem"}}
+          "In order to allow other users of this workspace to run this method, you will need
```

Comment:
If a method config has been contributed back to the methods repo, does it make sense to refer to a specific workspace?